.github/workflows/release.yaml

```
@@ -0,0 +1,57 @@
+  artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Code
+        uses: actions/checkout@v2
+      - name: Get Rpmbuild
+        run: sudo apt install rpm -y
+      - name: Generate Artifacts
+        run: make rpm
+      - name: Get Artifacts
+        id: get_artifacts
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)